core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -1440,6 +1443,26 @@ private CompiledQuery createTable(final ExecutionModel model, SqlExecutionContex
         }
     }
 
+    private void copyTableReaderMetadataToCreateTableModel(SqlExecutionContext executionContext, CreateTableModel model) throws SqlException {
```

Comment:
nit: this method should also clean up `likeTableName` field of the model, so that it's fully re-written.

Reply:
likeTableName is reset to null at last line of the method.